src-docs/src/views/card/card.js

```diff
@@ -17,7 +17,7 @@ const cardNodes = icons.map(function (item, index) {
         title={`Elastic ${item}`}
         isDisabled={item === 'Kibana' ? true : false}
         description="Example of a card's description. Stick to one or two sentences."
-        onClick={() => {}}
+        href="https://elastic.co"
```

Comment:
Let's keep them in the docs... 😉 
```suggestion
        href="#"
```